src/Symfony/Bridge/Twig/DataCollector/TwigDataCollector.php

```diff
@@ -62,24 +62,26 @@ public function lateCollect()
         $this->data['profile'] = serialize($this->profile);
         $this->data['template_paths'] = array();
 
-        $templateFinder = function (Profile $profile) use (&$templateFinder) {
-            if ($profile->isTemplate()) {
-                try {
-                    $template = $this->twig->load($name = $profile->getName());
-                } catch (\Twig_Error_Loader $e) {
-                    $template = null;
+        if (null !== $this->twig) {
```

Comment:
`if (null === $this->twig) {return;}`?
about the deprecation, too late IMHO if we want to be strict about the feat freeze